schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-contact.xsl

```diff
@@ -51,7 +51,7 @@
 
     <xsl:choose>
       <xsl:when test="$dcatElementConfig">
-        <xsl:for-each-group select="$allIndividualOrOrganisationWithoutIndividual" group-by="cit:name">
+        <xsl:for-each-group select="$allIndividualOrOrganisationWithoutIndividual" group-by="(cit:name,'individualDoesNotCarryACitName')[1]">
```

Comment:
@cmangeat can you explain this change?

Reply:
indeed, when grouping by something empty (such as cit:name for an individual), it is not looping, cf. [group-by-expression](https://stackoverflow.com/questions/79125480/how-to-handle-elegantly-an-empty-group-by-expression).

the workaround explained [here](https://www.mhonarc.org/archive/html/xsl-list/2014-01/msg00058.html) can be used.